packages/components/src/components/accordion/_accordion.scss

```diff
@@ -175,6 +184,54 @@
   }
 }
 
+@function px($value) {
+  @if unit($value) == 'px' {
+    @return $value;
+  }
+  @if unit($value) == 'rem' {
+    @return ($value / ($value * 0 + 1)) * 16px;
+  }
+  @error "Unable to convert unit #{unit($value)} to px";
+}
+
+@function rem($value) {
+  @if unit($value) == 'rem' {
+    @return $value;
+  }
+  @if unit($value) == 'px' {
+    @return ($value / 16px) * 1rem;
+  }
+  @error "Unable to convert unit #{unit($value)} to px";
```

Comment:
Should this say `unable to convert ... to rem`?

Reply:
Also, I get what this is doing, but how come we didn't just use our carbon rem function? Just wanna understand.